src/form/GooglePlaces.jsx

```diff
@@ -47,6 +48,7 @@ export default function GooglePlaces({
 
             // Check if address is valid
             if (address) {
+                setShouldScrollToTop(false);
```

Comment:
Instead of setting `scrollToTop` in this embedded component, I think I'd prefer that you move this check to `fieldOnChange` and check the `trigger` property to determine if Google Maps caused the field change. That way we can keep the `scrollToTop` checks in roughly the same place